app-misc/qlcplus/qlcplus-4.12.1.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils udev virtualx xdg
+
+DESCRIPTION="A software to control DMX or analog lighting systems"
+HOMEPAGE="https://www.qlcplus.org/"
+SRC_URI="https://github.com/mcallegari/${PN}/archive/QLC+_${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/qlcplus-QLC-_${PV}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	|| ( dev-embedded/libftdi:1 dev-embedded/libftdi:0 )
+	dev-libs/glib:2
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtmultimedia:5[widgets]
+	dev-qt/qtnetwork:5
+	dev-qt/qtscript:5
+	dev-qt/qtwidgets:5
+	media-libs/alsa-lib
+	media-libs/libmad
+	media-libs/libsndfile
+	sci-libs/fftw:3.0
+	virtual/libusb:0
+	virtual/libusb:1
+	virtual/udev
+"
+DEPEND="${RDEPEND}
+	dev-qt/qttest:5"
+
+src_prepare() {
+	default
+	sed -e "s:/etc/udev/rules.d:${EROOT}lib/udev/rules.d:" -i \
```

Comment:
In EAPI 7, EROOT no longer carries implicit slash. Also I don't think EROOT is valid here. What is the path used for?

Reply:
It's a leftover from an old solution to install the udev rules to the correct location.
Since this is now handled via the following lines modifying the contents of variables.pri, this sed command is no longer used and I'll remove it in the next commits

Reply:
Done